docs/general/staking-dashboard.md

```diff
@@ -0,0 +1,257 @@
+  show the role you currently have in staking (in this case active in both [Pools](#pools) and
+  [Nominate](#nominate)). You can also change the [network](#network) (currently on Kusama).
+
+- **Section B: The Accounts Panel** allows you to connect one account to the dashboard. Once
+  connected, the account will show up next to the `Accounts` button. If you are active as a
+  nominator and/or a pool member you will be able to see it here. In this case the account KSM Stash
+  is a nominator (with no controller) and a member of the pool Insight Finance. Controller accounts
+  are also shown here if applicable. Note that the dashboard will automatically fetch the stash or
+  the controller as read-only account if you connect one or the other.
+
+Note that Sections A and B will always be visible while you use the dashboard.
+
+- **Section C: The Stats Panel** shows the general view about current staking metrics including the
+  historical reward rate (including after
+  [inflation](../learn/learn-staking-advanced.md#inflation)), the supply staked and time remaining
+  in the current era.
+
+- **Section D: The Summary Panel** shows your current situation and gives you general tips about
+  staking. In this case the KSM Stash account is a nominator and a pool member, and by clicking on
+  `Manage >` you can go directly to the [Nominate](#nominate) and [Pools](#pools) pages,
+  respectively. There you can take actions such as changing staking preferences, bonding more funds,
+  etc.
+
+- **Section E: The Balance Panel** shows the bonded amount distinguishing between "Nominating", "In
+  a Pool", and "Not Staking". The amount that is not staking is further divided into "Free" (balance
+  that can be transferred) and "Reserve" (balance that is needed the keep the account on-chain, see
+  [existential deposit](../learn/learn-accounts.md#existential-deposit-and-reaping)). In this case
+  0.301 KSM are bonded for nominating, 0.3 KSM are bonded in a pool, and 0.145 KSM are not used for
+  staking. Of the non-staking balance 0.144 KSM are free while
+  {{ polkadot: <RPC network="kusama" path="consts.balances.existentialDeposit" defaultValue={333000000} filter="humanReadable"/> :polkadot }}
+  are reserved for the existential deposit.
+
+- **Section F: The Recent Payouts Panel** shows a bar chart with the rewards that have been paid out
+  to you in the past 15 days either as a nominator or a pool member (manually claimed). Note how the
+  bar of the 4th of April has the tip that is jot filled with color. This is to show a pending
+  payout (for nominators only). Below the bar chart there is a line chart showing then 10-day moving
+  average.
+
+Additional statistics can be found at the bottom of the overview page. Note also the current
+exchange rate for the native token {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} at the
+footer of the dashboard. This information is used to estimate how much your balance is worth.
```

Comment:
The exchange rate is not shown on staking.polkadot.network, only in development and staging on Github. Therefore it may be best to remove this sentence as to not confuse users.